patch.yml

```diff
@@ -13,6 +13,54 @@
 # limitations under the License.
 
 ---
+- name: Play | Set correct RDBMS release and set fact if this is a single node inventory
+  hosts: dbasm
+  gather_facts: false
+  tasks:
+    - name: include defaults
+      include_vars:
+        file: roles/common/defaults/main.yml
+
+    - name: printing oracle_rel
+      debug:
+        msg:
+          - "oracle_rel b4 is : {{ oracle_rel }}"
+
+    - name: printing gi_patches piped into json_query
+      debug:
+        msg:
+          - "Loop variables are: {{ gi_patches | json_query('[?base==`' + oracle_ver + '`].release') }}"
+
+    - name: Get the highest version number from gi_patches array
```

Comment:
This code looks like a copy from https://github.com/google/bms-toolkit/blob/e1a7eba1e1241231d9831252a2251b8928dc0f0e/roles/rac-db-setup/tasks/rac-db-install.yml#L46 ; it might help to actually copy it verbatim for consistency, including the comments, or even to put into its own playbook so it can be re-used :-)

Reply:
It was indeed taken from there and tested to populate the variable `highest_version_num` which is used in downstream roles. Will add a comment to reflect the same and not refactor at this point.

Reply:
Had responded in that branch.
Hope we can mark this with "Resolve conversation"